evenDistributionScore.py

```diff
@@ -64,12 +63,131 @@ def breakPattern(length):
         section = np.linspace(int(score[-1]), endpoint, num)
         score = np.concatenate((score, section))
 
-    score = np.concatenate((score, np.zeros(fiveMins))) # add final 5 minutes of zero score
+    score = np.concatenate((score, np.zeros(fiveMins)))  # add final 5 minutes of zero score
 
     return score
 
-        
-# Plot distibution to visualise
-x=np.linspace(0, getLength(), getLength())
+#Plot distibution to visualise
+x = np.linspace(0, getLength(), getLength())
 plt.plot(x, breakPattern(getLength()))
 plt.show()
+
+
+# Timecode plot
+maxTimes = [12.29, 25.55, 33.05, 49.15, 63.25, 77.05, 91.05, 105.05, 119.0, 123.05, 137.05]
+
+def getPrefNoBreakpoints(length):  # assuming continuous programme
```

Comment:
We have both created a function to do this so maybe should be combined
(mine just deals in frames instead of minutes)

Reply:
How would you combine them

Reply:
choose one to use and both use it. If you need seconds, just divide frames by 25 or vice versa